proto/tendermint/abci/types.proto

```diff
@@ -129,6 +129,8 @@ message RequestPrepareProposal {
   google.protobuf.Timestamp time = 6 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
   // Merkle root of the next validator set.
   bytes next_validators_hash = 7;
+  // Round number for the block.
+  int32 round = 8;
```

Comment:
WDYT usage `1XX` position for `round`, the same as for other dash related fields?